core/src/cluster_info_repair_listener.rs

```diff
@@ -276,6 +278,10 @@ impl ClusterInfoRepairListener {
             epoch_schedule.get_stakers_epoch(repairee_epoch_slots.root);
         let max_confirmed_repairee_slot =
             epoch_schedule.get_last_slot_in_epoch(max_confirmed_repairee_epoch);
+
+        // Skip the first slot in the iterator b/c we know it's the root slot which the repairee already
```

Comment:
nit: `b/c` -> `because` (more readable for non-native speakers)
nit2: `it's` -> `its`

Reply:
agree on the first, second should be it's no? xD